build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -234,15 +241,8 @@ private void GenerateRuntimeGraph(DotNetCli dotnetCli, string destinationDeps)
             }
         }
 
-        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot, string hostFxrVersion)
+        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot)
         {
-            File.Copy(
-                Path.Combine(_corehostLockedDirectory, HostArtifactNames.DotnetHostBaseName),
```

Comment:
We put the host in the shared framework in v1 because that was how you specified that you wanted to run the application on a specific shared framework. `C:\Program Files\dotnet\shared\Microsoft.NETCore.App\1.0.1\dotnet.exe` to ensure the app was using the 1.0.1 shared fx.

Do we no longer need this?  Can the version be specified through the command line args?